tests/test_integration.py

```diff
@@ -1,31 +1,38 @@
-import os.path as op
+from pathlib import Path
+import platform
 import shutil
 
 import pytest
 
+import cython_setuptools
 
-this_dir = op.dirname(__file__)
+this_dir = Path(__file__).parent
+cython_setuptools_path = (Path(cython_setuptools.__file__).parent).parent
 
 
-@pytest.mark.slow
-def test_compile_and_run_no_cythonize_mode(virtualenv, tmp_dir):
-    setup_path, pypkg_dir = _setup_source('setup-no-cythonize.py', tmp_dir)
-    virtualenv.run('CYTHONIZE=1 pip install -e %s' % pypkg_dir)
-    assert virtualenv.run('python -m bar', capture=True) == '2\n'
+def _setup_source(setup_name, tmp_dir: Path):
+    pypkg_dir = tmp_dir / "pypkg"
+    src_dir = tmp_dir / "src"
+    setup_path = pypkg_dir / "setup.py"
+    shutil.copytree(this_dir / "pypkg", str(pypkg_dir))
+    shutil.copytree(this_dir / "src", str(src_dir))
+    setup_path.symlink_to(pypkg_dir / setup_name)
+    return setup_path, pypkg_dir
+
+
+@pytest.mark.skipif(
+    platform.system().lower() == "windows",
+    reason="Windows environment variable are not easy to manage",
+)
+def test_compile_and_run_external_cythonize_mode(virtualenv, tmp_dir):
+    _, pypkg_dir = _setup_source("setup-no-cythonize.py", tmp_dir)
+    virtualenv.run(f"pip install {cython_setuptools_path}")
+    virtualenv.run(f"export CYTHONIZE=1; pip install -e {pypkg_dir.resolve()}")
+    assert int(virtualenv.run("python -m bar", capture=True)) == 2
 
 
-@pytest.mark.slow
```

Comment:
the pytest marker is removed in a specific commit.

Reply:
How slow is this test ?

Reply:
pytest -n 6 tests -> 6 passed in `14.97s`

pytest  tests -> 6 passed in `20.08s`

Reply:
but basically it's the same test as `test_compile_and_run_cythonize_mode ` but use the env variable instead of  setting cythonize to `True`  `setup(__file__, cythonize=True)`. 

I'm wondering if we really need to recompile all the bindings just to check the read of the env var CYTHONIZE ?